lib/_http_outgoing.js

```diff
@@ -120,6 +123,17 @@ function OutgoingMessage() {
 }
 util.inherits(OutgoingMessage, Stream);
 
+Object.defineProperty(OutgoingMessage.prototype, '_writableState', {
+  get() {
+    var state = this[kWritableState];
+    if (!state) {
+      state = new Writable.WritableState();
```

Comment:
If this property is useless we might as well just go with

```js
OutgoingMessage.prototype._writableState = new Writable.WritableState();
```

It’s weird but it should work and it avoid the hidden class change here.